docs/content/introduction/installation.md

```diff
@@ -1,10 +1,22 @@
 # Installation
 
-## Basic installation
+## Dependencies
 
-```{note}
-Before proceeding, make sure you have a [working installation of Docker](https://docs.docker.com/engine/install/) and a modern Python installation (Python 3.10+), ideally in a [virtual environment](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/).
-```
+Tesseract Core depends on Python 3.10+ (ideally installed within a [virtual environment](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/)) and Docker.
+
+[Docker Desktop](https://www.docker.com/products/docker-desktop/) makes it easy to get up and running with `docker`.
+It's available for macOS, and ships with several `docker` plugins needed by Tesseract Core.
+
+Docker Desktop is available on Linux for Debian and Fedora based distros.
+If you are on any other distribution of Linux, or prefer to minimally install the [`docker` engine CLI](https://docs.docker.com/engine/install/) via your package manager, you will need to install the plugins in addition to `docker`.
+These are
+
+1. [`docker-buildx`](https://github.com/docker/buildx)
+2. [`docker-compose`](https://github.com/docker/compose)
+
+though their names may differ between package manager repositories.
```

Comment:
I suggest we keep the original `{note}` box (but update the link to point to Docker Desktop instead of Docker engine), then add the quoted text *after* the basic installation instructions. Basically, we provide a minimal set of commands to run up-front to allow most people to jump right in, then follow up with more targeted instructions for cases in which that is expected to fail.